ci/tests/fputil.bash

```diff
@@ -4,9 +4,10 @@
 #
 # Helpers to test via FP util
 ####
+set -e
 export FPUTIL_TARGETS=("generate" "generate --ut" "build" "build --all" "check --all")
-export FPUTIL_DEPLOYS="${FPRIME_DIR} ${FPRIME_DIR}/Ref ${FPRIME_DIR}/RPI"
-
+# export FPUTIL_DEPLOYS="${FPRIME_DIR} ${FPRIME_DIR}/Ref ${FPRIME_DIR}/RPI"
+export FPUTIL_DEPLOYS="${FPRIME_DIR}/${TEST}"
```

Comment:
In your CI setup, you get the following errors in the Framework test:

 ```
/github/workspace/ci/tests/../tests/fputil.bash: line 28: cd: /github/workspace/Framework/: No such file or directory
```

Perhaps you should just set this variable in each test directly and remove this here?  e.g. in the Framework.bash add a line:

```
export FPUTIL_DEPLOYS="${FPRIME_DIR}"
```

Reply:
I replaced {FPUTIL_DEPLOYS} with  {FPRIME_DIR}, within the loop to make it more simple. The errors went away with this change.

Reply:
@r9-pena why not do something similar to RPI and Ref:

In Ref.bash:
```
fputil_action "${FPRIME_DIR}/Ref" "${target}"
```

In RPI.bash:
```
fputil_action "${FPRIME_DIR}/RPI" "${target}"
```

Reply:
Added the following line below, as originally requested, to maintain consistency across all bash files.
```
export FPUTIL_DEPLOYS="${FPRIME_DIR}"
export FPUTIL_DEPLOYS="${FPRIME_DIR}/Ref"
export FPUTIL_DEPLOYS="${FPRIME_DIR}/RPI"
```